test/util.ts

```diff
@@ -1,116 +1,70 @@
-import 'mocha';
-import { expect } from 'chai';
-
 export function expectAnOption(val) : void {
-  expect(val)
-    .to.be.a('object');
-  expect(val)
-    .to.have.property('isSome').that.is.a('function');
-  expect(val)
-    .to.have.property('isNone').that.is.a('function');
-  expect(val)
-    .to.have.property('toString').that.is.a('function');
-  expect(val)
-    .to.have.property('expect').that.is.a('function');
-  expect(val)
-    .to.have.property('unwrap').that.is.a('function');
-  expect(val)
-    .to.have.property('unwrapOr').that.is.a('function');
-  expect(val)
-    .to.have.property('unwrapOrElse').that.is.a('function');
-  expect(val)
-    .to.have.property('map').that.is.a('function');
-  expect(val)
-    .to.have.property('mapOr').that.is.a('function');
-  expect(val)
-    .to.have.property('mapOrElse').that.is.a('function');
-  expect(val)
-    .to.have.property('and').that.is.a('function');
-  expect(val)
-    .to.have.property('flatMap').that.is.a('function');
-  expect(val)
-    .to.have.property('or').that.is.a('function');
-  expect(val)
-    .to.have.property('orElse').that.is.a('function');
-  expect(val)
-    .to.have.property('match').that.is.a('function');
-  expect(val)
-    .to.have.property('clone').that.is.a('function');
-  expect(val)
-    .to.have.property('filter').that.is.a('function');
-  expect(val)
-    .to.have.property('forEach').that.is.a('function');
-  expect(val)
-    .to.have.property('equals').that.is.a('function');
-  expect(val)
-    .to.have.property('hasValue').that.is.a('function');
-  expect(val)
-    .to.have.property('contains').that.is.a('function');
+  expect(val).toHaveProperty('isSome'); expect(val.isSome).toBeInstanceOf(Function);
```

Comment:
do you intend for these to be on the same line? can you not chain the expectations with jest?

Reply:
I did mean for them to be on the same line just for visibility's sake.  I agree it's not the prettiest code.  I tried chaining, but I couldn't quite find what I needed, and Jest's docs weren't very verbose, so I just went with this for now.